### PR TITLE
feat: Wire in hiscores to boss panel and add renown costs

### PR DESCRIPTION
**Pull request by DMills94**
The husky file update has no actual code change, but it resolved my issue on my work laptop for the pre-commit not running. Shouldn't be an issue!

### CODE REVIEW

File: src/components/ManageDataModal.js [hunk 1]
```diff
@@ -280,8 +280,10 @@ function ResetDataModalContent({ setIsOpen }) {
                     onClick={() => {
                         resetLocalStorageData();
                         setSuccessText('Data has been deleted. Reloading page...');
+                        const currentParams = new URLSearchParams(window.location.search);
+                        currentParams.delete('open');
                         setTimeout(() => {
-                            window.location.reload();
+                            window.location.search = currentParams.toString();
```
Comment (DMills94) [inline src/components/ManageDataModal.js:286]: Someone added a query param to keep modal windows open 😠

File: src/components/BossesPanel.js [hunk 2]
```diff
@@ -1,181 +1,109 @@
...
                     <>
-                        {/* TODO add sage's renown values for unlocks */}
-                        {/* <span className='heading-accent-md'>{selectedBoss}</span>
-                        <p className='italic text-sm'>Cost: 500 league points</p> */}
+                        <p className='italic text-sm'>
+                            Kills: {numberWithCommas(hiscores[selectedBoss.hiscoresName].score)}
+                        </p>
+                        <p className='italic text-sm'>
+                            Rank: {numberWithCommas(hiscores[selectedBoss.hiscoresName].rank)}
+                        </p>
```
Comment (chaiinchomp) [inline src/components/BossesPanel.js:25]: do we need any fallbacks here if you don't have any kc? Or does it have a default already

File: src/store/unlocks/constants.js [hunk 3]
```diff
@@ -1,20 +1,10 @@
-export const CURRENT_VERSION = 3;
+export const CURRENT_VERSION = 4;
```
Comment (chaiinchomp) [inline src/store/unlocks/constants.js:1]: 👏👏

Comment (chaiinchomp) [reply src/store/unlocks/constants.js:1]: If you want to have this effect existing state, you'll need to add a version update function as well - like if you wanted to remove the default bosses from users' unlock lists

File: src/components/BossesPanel.js (hunk 2 above)
Comment (DMills94) [reply src/components/BossesPanel.js:25]: Fair point, yes i expect it will show -1 so i'll change it to be a `-`

**Pull request merged**